### ISSUE TITLE
Bears' configs

### ISSUE THREAD

**Issue by Mariatta** [coala/coala#3255]
_From @Asnelchristian on October 11, 2016 14:38_

It will be good to have a page ( or maybe we should just do this under `writing bears` docs) that explain how to name settings, with some common examples.

_Copied from original issue: coala/documentation#181_

**Comment by marian-ald** [comment]
@Mariatta  you mean writing a document where to explain the names of the bears?

**Comment by jayvdb** [comment]
> @Mariatta you mean writing a document where to explain the names of the bears?

No.  Naming of settings, e.g. that appear in `.coafile`.  See https://github.com/coala/coala-bears/issues/1112#issuecomment-267953755 for an example.
